.changeset/sour-cooks-repeat.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/backend-common': patch
+---
+
+Renamed the `loadBackendConfig` option `additionalConfig` to `additionalConfigs` as an array, and ensured that they get passed on properly.
+
+This is technically breaking, but the [original addition](https://github.com/backstage/backstage/pull/16643) hasn't been released in mainline yet so we are taking this step now as a `patch` change.
```

Comment:
Could we change the existing changeset in the PR mentioned for tomorrows release and skip this file entirely?

Reply:
Is that kosher after -next releases have been out? Maybe?